backend/app/assets/javascripts/spree/backend/global/ransack.js

```diff
@@ -1,11 +1,14 @@
 /* eslint-disable no-var */
 document.addEventListener('DOMContentLoaded', function() {
   var QuickSearchInput = document.getElementById('quick_search')
-  var QuickSearchPlaceHolder = QuickSearchInput.placeholder
-  var TargetSearchFieldId = document.querySelector('input.js-quick-search-target').id
-  var AssociatedLabelName = document.querySelector(`label[for="${TargetSearchFieldId}"]`).innerHTML
 
-  QuickSearchInput.placeholder = `${QuickSearchPlaceHolder} ${AssociatedLabelName}`
+  if (QuickSearchInput) {
+    var QuickSearchPlaceHolder = QuickSearchInput.placeholder
+    var TargetSearchFieldId = document.querySelector('input.js-quick-search-target').id
+    var AssociatedLabelName = document.querySelector(`label[for="${TargetSearchFieldId}"]`).innerHTML
```

Comment:
@MatthewKennedy ``label[for="${TargetSearchFieldId}"]`` this notation requires ES6 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)  so we need to rename this file from `ransack.js` to `ransack.es6`

Reply:
@damianlegawiec I'll update that.

Reply:
@damianlegawiec Shall I update the other newly created files in the` global` folder to `.es6`, as they are all new files moving forward to 4.3 and so, shouldn't create any backward compatibility issues.

Reply:
@MatthewKennedy I think we should start using ES6 for any new stuff from now on, it will make our life easier :)

Reply:
@damianlegawiec Much easier, I have no problem with that.  :)